desktop/libs/indexer/src/indexer/indexers/sql.py

```diff
@@ -269,7 +258,7 @@ def create_table_from_a_file(self, source, destination, start_time=-1, file_enco
           extra_create_properties += "\nTBLPROPERTIES('transactional'='true', 'transactional_properties'='%s')" % \
               default_transactional_type
 
-      sql += '''\n\nCREATE TABLE `%(database)s`.`%(final_table_name)s`%(comment)s
+      sql += '''\n\nCREATE %(table_type)sTABLE `%(database)s`.`%(final_table_name)s`%(comment)s
```

Comment:
Is this correct where there is no space between %(table_type)sTABLE?

Reply:
Yes, added space in 'EXTERNAL ' word. I added there as if table_type is empty then there will be 2 spaces and we don't want that.